QUARKUS-1415.md

```diff
@@ -0,0 +1,28 @@
+# QUARKUS-1415 Reactive REST Client improvements
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1415
+
+Test plan covers RESTEasy Reactive functionality provided by `quarkus-resteasy-reactive` and `quarkus-rest-client-reactive` extensions,
+with emphasis on multipart form data support.
+
+https://quarkus.io/version/main/guides/resteasy-reactive
+
+https://quarkus.io/version/main/guides/rest-client-reactive
+
+## Related test plans
+- QUARKUS-1075
+
+## Scope of the testing
+- Upload and download of large files
+- Support for proxy authentication
```

Comment:
How do you plan to simulate proxy?

Reply:
Any concrete proxy? If so please include its name / image tag here

Reply:
Nothing in particular. Any recommendations?

Reply:
No, you are preparing the plan, so I was wondering if you have something.